http.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
```

Comment:
And let's use our own parser jsonx

Reply:
I was using `encoding/json` because I needed to manipulate JSON (mostly combine), not only parse. But I agree, having two separate parsers is a bit excessive.